rom/laghos_rom.cpp

```diff
@@ -1379,6 +1360,17 @@ void ROM_Basis::ReadSolutionBases(const int window)
         MFEM_VERIFY(basisX->numRows() == tH1size, "");
         basisV = basisX;
     }
+
+    if (!use_sns)
```

Comment:
It would be easier to read if the cases were reversed, with `if (use_sns)`.

Reply:
@dylan-copeland Thanks for the suggestion. The cases are reversed.